### ISSUE TITLE
Images being rotated

### ISSUE THREAD

**Issue by andrecalil** [feimosi/baguetteBox.js#164]
Hi there,

First off, thanks for this great tool.

Now, to the problem we're having: I've noticed that, if the image is larger then the screen, it gets rotated (sometimes clockwise, sometimes not). Both thumbnail and large view are rotated.
If I grab the src link and go straight to the file in question, the orientation is correct.

Is there a way to prevent baguetteBox from doing it? Maybe resizing the image on the fly, keeping the original proportion?

Thanks in advance!

**Comment by feimosi** [comment]
Hi @andrecalil,
If even the thumbnail gets rotated that's very unlikely it's baguetteBox's fault. Does it also happen for you on the demo page? Could you provide me with some live example?

**Comment by andrecalil** [comment]
Hey @feimosi , sorry for not getting back to you earlier.

Yesterday I was putting together an example for you and found something interesting. I opened a picture on Fireworks to edit some sensitive info and, to my surprise, it was rotated to the left. On Windows default "photo viewer" app, it was on the proper orientation (portrait).
I started putting together the page and, even without baguette, the photo was rotated.

After some quick search, I found this issue: https://stackoverflow.com/questions/24658365/img-tag-displays-wrong-orientation
If you read through the comments, you'll see reports from Samsung users. Funny enough, the photo I was playing with was taken from a Samsung phone.

Now I'll look for a tool to remove EXIF information and will try again. I'm closing this issue for now, since it does not seem to be accurate after all.

Thank you for your time.
